.github/ISSUE_TEMPLATE/distributors-application.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Distributors Application
```

Comment:
If a company or organisation uses Kurator, we should inform them of the Kurator security vulnerability before releasing the vulnerability. To avoid someone maliciously using the Kurator vulnerability for attack. `Distributors` these companies or organisations